eng/scripts/automation/Invoke-CadlDataPlaneGenerateSDKPackage.ps1

```diff
@@ -52,8 +52,37 @@ param (
   [string]$specRoot = "",
   [string]$repo="",
   [string]$commit="",
-  [string]$additionalSubDirectories="" #additional directories needed, separated by semicolon if more than one
+  [string]$additionalSubDirectories="", #additional directories needed, separated by semicolon if more than one
+  [switch]$help
 )
+if ($help) {
+  Write-Host("Usage:
+Invoke-CadlDataPlaneGenerateSDKPackage.ps1 -service <servicename> -namespace Azure.<group>.<service> -sdkPath <sdkrepoRootPath> -cadlRelativeFolder <relativeCadlProjectFolderPath> [-commit <commitId>] [-repo <specRepo>] [-specRoot <specRepoRootPath>] [-additionalSubDirectories <relativeFolders>]
+
+Options:
+-service  [Required] takes Azure client service directory name in sdk repo. e.g. AnomalyDetector
+-namespace  [Required] takes the namespace of the Azure client library.
+-sdkPath  [Required] takes the address of the root directory of sdk repo. e.g. /home/azure-sdk-for-net
+-cadlRelativeFolder [Required] takes the relative path of the cadl project folder in spec repo. e.g. specification/cognitiveservices/AnomalyDetector
+-additionalSubDirectories [Optional] takes the relative paths of the additional directories needed by the cadl project
+-commit takes the git commit hash  (e.g. ac8e06a2ed0fc1c54663c98f12c8a073f8026b90)
+-repo takes the `<owner>/<repo>` of the REST API specification repository. (e.g. Azure/azure-rest-api-specs)
+-specRoot takes the file system path of the spec repo. e.g. /home/azure-rest-api-specs
```

Comment:
we shouldn't need this since `Cadl-Project-Sync.ps1` will pull all of this in with a sparse-checkout

Reply:
If customer want to use a local specs repo that is already present, he can specify `spec-root-dir` to provide the path of the root directory of the specs repo, then `Cadl-Project-Sync.ps1` will not parse-checkout the cadl project remotely, and just use this local cadl folder and copy it to the `TempCadlFiles` for generation.

Reply:
I was also wondering the same thing here. We want to be able to support an existing clone of the specs repo but I don't see how this script is currently handling that. Do you expect folks to either already have a manual clone or have called Cadl-Project-Sync before calling this script?

Reply:
If you have already a manual cloned spec repo, you need to specify `spec-root-dir` parameter when run `Cadl-Project.Sync.ps1`, then the script will not parse-checkout the cadl project, and just copy the existing cadl folder in the cloned swagger repo to `TempCadlFiles`

Reply:
Customer using the local dir is an unsupported scenario and we shouldn't advertise it until we fully implement it.  I realize that in a few rare cases it can work but its not a fully vetted scenario.

Reply:
still need to remove the -specRoot mention

Reply:
removed